requirements.txt

```diff
@@ -33,6 +33,7 @@ django-storages==1.7.1
 django-taggit==0.23.0     # via wagtail
 django-templated-mail==1.1.1  # via djoser
 django-treebeard==4.3     # via wagtail
+django-user-agents==0.4.0
```

Comment:
I am not sure why we update this file ? As per my knowledge, This file is autogenerated by pip-compile.

Reply:
you are right, I didn't updated this file manually. I ran pip-compile and this file is updated automatically.